### DISCUSSION TITLE
SPI support on NXP´s imxrt1062 SOC

### DISCUSSION THREAD

**Discussion by bdkrae** [zephyrproject-rtos/zephyr#31542] [Q&A] [answered]
Is there a reason why the NXP imxrt SOC series does not support SPI (called LPSPI on this series) on some devices? 

The imxrt1061, imxrt1062, imxrt1064 devices are missing the ```select HAS_MCUX_LPSPI``` line the [Kconfig.soc](https://github.com/zephyrproject-rtos/zephyr/blob/master/soc/arm/nxp_imx/rt/Kconfig.soc). If this config is added, the lpspi seems to work fine on first view.

This question is related to [this](https://github.com/zephyrproject-rtos/zephyr/issues/30204#issuecomment-764974199) post in #30204.

**Comment by hendriklehmkuhl** [comment, marked as answer]
Hi @bdkrae,

have a look on this closed issue: [#26428](https://github.com/zephyrproject-rtos/zephyr/issues/26428)

**Reply by bdkrae** [reply]
Many thanks, this answers my question!
